eth/watchers/orchestratorwatcher.go

```diff
@@ -0,0 +1,135 @@
+package watchers
+
+import (
+	ethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/golang/glog"
+	"github.com/livepeer/go-livepeer/common"
+	"github.com/livepeer/go-livepeer/eth"
+	"github.com/livepeer/go-livepeer/eth/blockwatch"
+	"github.com/livepeer/go-livepeer/eth/contracts"
+)
+
+type OrchestratorWatcher struct {
+	db      OrchestratorStore
```

Comment:
Could we name this something like `store` since the name `db` feels strongly tied to a database and the interface is not necessarily implemented by a database?

Reply:
fixed in 6a397072